diff --git a/source/ConversationPanel.cpp b/source/ConversationPanel.cpp
index 1fa867cb89ec..4fa1aec40298 100644
--- a/source/ConversationPanel.cpp
+++ b/source/ConversationPanel.cpp
@@ -145,7 +145,9 @@ void ConversationPanel::Draw()
 			// Handle mouse clicks in whatever field is not selected.
 			if(side != choice)
 			{
-				AddZone(Rectangle(center, fieldSize), [this, side](){ this->ClickName(side); });
+				AddZone(Rectangle(center, fieldSize), [this, side](){
+					SDL_StartTextInput(); // trigger the keyboard if needed
+					this->ClickName(side); });
 				continue;
 			}
 
diff --git a/source/Dialog.cpp b/source/Dialog.cpp
index 7303d31a3260..da406e226def 100644
--- a/source/Dialog.cpp
+++ b/source/Dialog.cpp
@@ -286,6 +286,12 @@ bool Dialog::Click(int x, int y, int clicks)
 		}
 	}
 
+	if (intFun || stringFun)
+	{
+		// Clicked on edit field. popup touchscreen keyboard if needed.
+		SDL_StartTextInput();
+	}
+
 	return true;
 }
 
