diff --git a/CMakeLists.txt b/CMakeLists.txt
index 65ecafdc..e4aebe9d 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -144,8 +144,20 @@ endif()
 
 # Link with OpenGL or OpenGL ES.
 if(ES_GLES)
-	find_package(OpenGL REQUIRED OpenGL EGL)
-	target_link_libraries(ExternalLibraries INTERFACE OpenGL::OpenGL OpenGL::EGL)
+	if(SAILFISHOS)
+		# our cmake doesn't support finding GLES stuff, lets do it manually:
+		find_package(PkgConfig REQUIRED)
+		pkg_check_modules(GLES REQUIRED glesv2)
+		target_link_libraries(ExternalLibraries INTERFACE "${GLES_LIBRARIES}")
+		target_include_directories(ExternalLibraries INTERFACE "${GLES_INCLUDE_DIRS}")
+		pkg_check_modules(EGL REQUIRED egl)
+		target_link_libraries(ExternalLibraries INTERFACE "${EGL_LIBRARIES}")
+		target_include_directories(ExternalLibraries INTERFACE "${EGL_INCLUDE_DIRS}")
+		#pkg_check_modules(WEGL REQUIRED wayland-egl)
+	else()
+		find_package(OpenGL REQUIRED OpenGL EGL)
+		target_link_libraries(ExternalLibraries INTERFACE OpenGL::OpenGL OpenGL::EGL)
+	endif()
 	target_compile_definitions(EndlessSkyLib PUBLIC ES_GLES)
 else()
 	# The Steam runtime can't rely on GLVND being available, so link against the legacy interface.
