diff --git a/source/GameWindow.cpp b/source/GameWindow.cpp
index e2d6e253..72131140 100644
--- a/source/GameWindow.cpp
+++ b/source/GameWindow.cpp
@@ -21,6 +21,9 @@ this program. If not, see <https://www.gnu.org/licenses/>.
 #include "opengl.h"
 #include <SDL2/SDL.h>
 
+#include <SDL2/SDL_syswm.h>
+#include <wayland-client-protocol.h>
+
 #include <cstring>
 #include <sstream>
 #include <string>
@@ -165,6 +168,14 @@ bool GameWindow::Init(bool headless)
 		return true;
 	}
 
+	// Rotate Window in SailfishOS
+	SDL_SysWMinfo swminfo;
+	SDL_VERSION(&swminfo.version)
+	SDL_GetWindowWMInfo(mainWindow, &swminfo);
+	wl_surface *sdl_wl_surface = swminfo.info.wl.surface;
+	SDL_Log("Rotating Wayland display");
+	wl_surface_set_buffer_transform(sdl_wl_surface, WL_OUTPUT_TRANSFORM_90);
+
 	// Settings that must be declared before the context creation.
 	SDL_GL_SetAttribute(SDL_GL_DOUBLEBUFFER, 1);
 #ifdef _WIN32
