diff --git a/CMakeLists.txt b/CMakeLists.txt
index 65ecafd..57c7b03 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -10,6 +10,11 @@ cmake_dependent_option(ES_USE_OFFSCREEN
 	"Use SDL's offscreen backend instead of Xvfb for the integration tests" OFF UNIX OFF)
 cmake_dependent_option(ES_CREATE_BUNDLE "Create a Bundle instead of an executable. Not suitable for development purposes." OFF APPLE OFF)
 
+option(SAILFISHOS "Use Sailfish OS specific settings. Turn off GLEW and use GLES." OFF)
+if(SAILFISHOS)
+	set(ES_GLES ON)
+endif()
+
 # Support Debug and Release configurations.
 set(CMAKE_CONFIGURATION_TYPES "Debug" "Release" CACHE STRING "" FORCE)
 
@@ -66,7 +71,9 @@ find_package(SDL2 CONFIG REQUIRED)
 find_package(PNG REQUIRED)
 find_package(JPEG REQUIRED)
 if(NOT APPLE)
-	find_package(GLEW REQUIRED)
+	if(NOT SAILFISHOS)
+		find_package(GLEW REQUIRED)
+	endif()
 endif()
 
 # Only very new versions of OpenAL provide a config package, so try to find it manually.
@@ -160,8 +167,10 @@ else()
 		# Apple deprecated OpenGL in MacOS 10.14, but we don't care.
 		target_compile_definitions(EndlessSkyLib PUBLIC GL_SILENCE_DEPRECATION)
 	else()
-		# GLEW is only needed on Linux and Windows.
-		target_link_libraries(ExternalLibraries INTERFACE GLEW::GLEW)
+		if (NOT SAILFISHOS)
+			# GLEW is only needed on Linux and Windows.
+			target_link_libraries(ExternalLibraries INTERFACE GLEW::GLEW)
+		endif()
 	endif()
 endif()
 
@@ -221,6 +230,7 @@ elseif(WIN32)
 else()
 	add_executable(EndlessSky source/main.cpp)
 	set_target_properties(EndlessSky PROPERTIES OUTPUT_NAME "endless-sky")
+	target_compile_definitions(EndlessSky PUBLIC ES_GLES)
 endif()
 
 # Link with the library dependencies.
