diff --git a/CMakeLists.txt b/CMakeLists.txt
index eabf82cb..20c9398d 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -334,7 +334,7 @@ elseif(WIN32)
 	include(CPack)
 elseif(UNIX)
 	# Install the binary.
-	install(TARGETS EndlessSky CONFIGURATIONS Release RUNTIME DESTINATION games)
+	install(TARGETS EndlessSky CONFIGURATIONS Release RUNTIME DESTINATION bin)
 
 	# Install the desktop file.
 	install(FILES io.github.endless_sky.endless_sky.desktop DESTINATION share/applications)
@@ -360,12 +360,12 @@ elseif(UNIX)
 	install(FILES ${CMAKE_CURRENT_BINARY_DIR}/endless-sky.6.gz DESTINATION share/man/man6)
 
 	# Install the resource files.
-	install(DIRECTORY data DESTINATION share/games/endless-sky)
-	install(DIRECTORY images DESTINATION share/games/endless-sky)
-	install(DIRECTORY shaders DESTINATION share/games/endless-sky)
-	install(DIRECTORY sounds DESTINATION share/games/endless-sky)
-	install(FILES credits.txt DESTINATION share/games/endless-sky)
-	install(FILES keys.txt DESTINATION share/games/endless-sky)
+	install(DIRECTORY data DESTINATION share/endless-sky)
+	install(DIRECTORY images DESTINATION share/endless-sky)
+	install(DIRECTORY shaders DESTINATION share/endless-sky)
+	install(DIRECTORY sounds DESTINATION share/endless-sky)
+	install(FILES credits.txt DESTINATION share/endless-sky)
+	install(FILES keys.txt DESTINATION share/endless-sky)
 	install(FILES copyright DESTINATION share/doc/endless-sky)
 	install(FILES changelog DESTINATION share/doc/endless-sky)
 	install(FILES license.txt DESTINATION share/doc/endless-sky)
diff --git a/source/Files.cpp b/source/Files.cpp
index 038c0093..e5215eec 100644
--- a/source/Files.cpp
+++ b/source/Files.cpp
@@ -128,12 +128,15 @@ void Files::Init(const char * const *argv)
 		// the end, so parent paths do not include it.
 		static const filesystem::path LOCAL_PATH = "/usr/local";
 		static const filesystem::path STANDARD_PATH = "/usr";
-		static const filesystem::path RESOURCE_PATH = "share/games/endless-sky/";
+		static const filesystem::path RESOURCE_PATH = "/home/.local/share/endless-sky/";
 
+		/*
 		if(IsParent(LOCAL_PATH, resources))
 			resources = LOCAL_PATH / RESOURCE_PATH;
 		else if(IsParent(STANDARD_PATH, resources))
 			resources = STANDARD_PATH / RESOURCE_PATH;
+		*/
+		resources = RESOURCE_PATH;
 #endif
 	}
 	// If the resources are not here, search in the directories containing this
