diff -Naur a/source/Preferences.cpp b/source/Preferences.cpp
--- a/source/Preferences.cpp	2023-06-18 23:53:14.000000000 +0200
+++ b/source/Preferences.cpp	2024-04-12 17:41:04.956675669 +0200
@@ -37,7 +37,7 @@
 	const string EXPEND_AMMO = "Escorts expend ammo";
 	const string FRUGAL_ESCORTS = "Escorts use ammo frugally";
 
-	const vector<double> ZOOMS = {.25, .35, .50, .70, 1.00, 1.40, 2.00};
+	const vector<double> ZOOMS = {.25, .35, .50, .70, 1.00, 1.40, 2.00, 2.25, 2.50, 3.00, 3.33, 3.5, 4.0};
 	int zoomIndex = 4;
 	constexpr double VOLUME_SCALE = .25;
 
